webpack.js

```diff
@@ -0,0 +1,35 @@
+const { merge } = require('webpack-merge')
+const webpackConfig = require('@nextcloud/webpack-vue-config')
+const BabelLoaderExcludeNodeModulesExcept = require('babel-loader-exclude-node-modules-except')
+
+const config = {
+	module: {
+		rules: [
+			{
+				test: /\.vue$/,
+				loader: 'vue-loader',
+				exclude: BabelLoaderExcludeNodeModulesExcept([
+					'@juliushaertl/vue-richtext',
+					'vue-material-design-icons',
+				]),
+			},
+			{
+				test: /\.js$/,
+				loader: 'babel-loader',
+				exclude: BabelLoaderExcludeNodeModulesExcept([
+					'@juliushaertl/vue-richtext',
+					'@nextcloud/event-bus',
+					'semver',
+				]),
+			},
+		],
```

Comment:
You could probably just add the exclude to the original `/.js$/` rule instead of addign it and removing duplicate.

Maybe I should create utils functions for this in webpack-vue-config :thinking:

Reply:
rules being an array without useful indexes is a pitty. But I nailed it